tests/Unit/EnvironmentTest.php

```diff
@@ -102,7 +102,7 @@ public function it_should_be_resolvable_from_string(string $type)
         $environment = Environment::fromString($type);
 
         $this->assertInstanceOf(Environment::class, $environment);
-        $this->assertEquals($type, $environment);
+        $this->assertSame($type, (string) $environment);
```

Comment:
What do you think about directly invoking `__toString()` here (and below) instead?

Reply:
I wouldn't mind doing that, but it just feels kinda dirty directly calling a magic method.

Reply:
As a matter of personal taste, I don't like calling magic-methods either. I think the cast is fine, but would also be okay adding a `toString()` method to `Environment`. Unsure what the benefit of that would be though.